libs/richtext/src/lib/to-html.tsx

```diff
@@ -1,11 +1,12 @@
 import {Node} from 'slate'
-import {slateToDomConfig, slateToHtml} from 'slate-serializers'
 import {BlockFormat} from './block-format'
 import {InlineFormat} from './inline-format'
 import {TextFormat} from './text-format'
 
-export const toHtml = (nodes: Node[]) =>
-  slateToHtml(nodes, {
+export const toHtml = async (nodes: Node[]) => {
+  const {slateToHtml, slateToDomConfig} = await import('slate-serializers')
```

Comment:
slate-serializers is quite a big package, no need to load it on every page even when this function isn't called.